### ISSUE TITLE
Error launching container

### ISSUE THREAD

**Issue by andrewschott** [linuxserver/docker-rutorrent#81]
Greetings!

Host OS: Centos7, RHEL7, and Fedora 27 WS all fail identically, docker pulled from RH supplied repos (appropriate to distro/release)

I am having a problem getting the container to work correctly.  Ages ago my script worked, now with newer versions it is not.  Here is the docker logs error that continually spews out:

> nginx: [emerg] open() "/config/nginx/nginx.conf" failed (2: No such file or directory)

And for reference, here is my script (the paths locally are correct, and I did recreate the volume sources fresh, and the appropriate user/group is indeed 1000:1000):

> #!/bin/bash
> 
>  docker run -d \
>         --sig-proxy=false \
>         --name=rutorrent \
>         -v /storage/docker/rutorrent/config:/config \
>         -v /storage/docker/rutorrent/downloads:/downloads \
>         -e PGID=1000 -e PUID=1000 \
>         -e TZ=America/Chicago \
>         -p 80:80 -p 5000:5000 \
>         -p 51413:51413 -p 6881:6881/udp \
>         linuxserver/rutorrent

Thanks!
Andrew.

**Comment by thelamer** [comment]
Check folder permissions: 

```
chmod 755 /storage/docker/rutorrent/config /storage/docker/rutorrent/downloads
chown 1000:1000 /storage/docker/rutorrent/config /storage/docker/rutorrent/downloads
```

To confirm it is your folders you can try running it once without the volumes to ensure it starts up properly: 

```
docker run -d \
--sig-proxy=false \
--name=rutorrent \
-e PGID=1000 -e PUID=1000 \
-e TZ=America/Chicago \
-p 80:80 -p 5000:5000 \
-p 51413:51413 -p 6881:6881/udp \
linuxserver/rutorrent
```

**Comment by andrewschott** [comment]
OK, thanks.

Here is what my problem WAS, and the fix.

SELinux was for some reason now having a shitfit over a ton of permissions.

I added:

> --privileged

to the docker run command, and things work perfectly.  Thanks.  Your idea of the permissions and no volume were the trigger to me firing up setroubleshoot to figure out what it hated.  Then I remembered that parameter.

Thanks again,
Andrew.
